hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/lock/OzoneManagerLock.java

```diff
@@ -188,13 +188,13 @@ private OMLockDetails acquireLock(Resource resource, boolean isReadLock,
     long startWaitingTimeNanos = Time.monotonicNowNanos();
 
     ReentrantReadWriteLock lock = getLock(resource, keys);
```

Comment:
What's the impact of removing these locks on correctness ?

Reply:
This is just for test purpose, to have parallelism